plugins.go

```diff
@@ -38,7 +39,7 @@ var (
 
 	// eventHooks is a map of hook name to Hook. All hooks plugins
 	// must have a name.
-	eventHooks = make(map[string]EventHook)
+	eventHooks = sync.Map{}
```

Comment:
Should this be a pointer 🤔 ? I know we don't pass this value around, but [all the methods are defined on `*sync.Map`](https://golang.org/pkg/sync/#Map). (I know it says the zero value is valid, but I just want to double-check.)

Reply:
No, this isn't necessary, please check this:
https://stackoverflow.com/a/47825590